src/sciwrappers.h

```diff
@@ -72,6 +72,7 @@ gchar				sci_get_char_at				(ScintillaObject *sci, gint pos);
 
 void				sci_scroll_caret			(ScintillaObject *sci);
 gint				sci_find_text				(ScintillaObject *sci, gint flags, struct Sci_TextToFind *ttf);
+void				sci_set_font_fract				(ScintillaObject *sci, gint style, const gchar *font, double size);
```

Comment:
there is one tab too many on the alignment on the right, we use tabs of width 4 (see HACKING file on the repository)